src/main/java/com/google/api/generator/gapic/composer/grpcrest/ServiceClientClassComposer.java

```diff
@@ -26,4 +40,37 @@ protected ServiceClientClassComposer() {
   public static ServiceClientClassComposer instance() {
     return INSTANCE;
   }
+
+  @Override
+  protected List<CommentStatement> createClassHeaderComments(
+      Service service,
+      TypeStore typeStore,
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes,
+      List<Sample> samples) {
+    TypeNode clientType = typeStore.get(ClassNames.getServiceClientClassName(service));
+    TypeNode settingsType = typeStore.get(ClassNames.getServiceSettingsClassName(service));
+    Sample classMethodSampleCode =
+        ServiceClientHeaderSampleComposer.composeClassHeaderSample(
+            service, clientType, resourceNames, messageTypes);
+    Sample credentialsSampleCode =
+        ServiceClientHeaderSampleComposer.composeSetCredentialsSample(clientType, settingsType);
+    Sample endpointSampleCode =
+        ServiceClientHeaderSampleComposer.composeSetEndpointSample(clientType, settingsType);
+    Sample transportSampleCode =
+        ServiceClientHeaderSampleComposer.composeTransportSample(
+            clientType, settingsType, "defaultHttpJsonTransportProviderBuilder");
+    samples.addAll(
+        Arrays.asList(
+            classMethodSampleCode, credentialsSampleCode, endpointSampleCode, transportSampleCode));
+
+    return ServiceClientCommentComposer.createClassHeaderComments(
+        service,
+        SampleCodeWriter.writeInlineSample(classMethodSampleCode.body()),
```

Comment:
You could also combine all the different sample.body into one List<Statement> and pass the whole list to SampleCodeWriter.writeInlineSample()